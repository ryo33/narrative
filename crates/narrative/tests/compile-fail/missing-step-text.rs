#[narrative::story("Story with invalid step")]
trait InvalidStepStory {
    #[step] // Missing required text parameter
    fn invalid_step(&self);
}

fn main() {}
