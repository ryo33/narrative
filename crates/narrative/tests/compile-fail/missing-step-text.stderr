error: expected a step, trait, struct, enum, or let
 --> tests/compile-fail/missing-step-text.rs:4:5
  |
4 |     fn invalid_step(&self);
  |     ^^
